scripts/createPage/builder.js

```diff
@@ -0,0 +1,17 @@
+const upperCaseFirstLetter = string => string.charAt(0).toUpperCase() + string.slice(1);
+
+const builder = {
+  buildJS: pageTitle => `import Head from 'components/head';
+
+export default () => {
+  const pageTitle = '${upperCaseFirstLetter(pageTitle)}';
```

Comment:
It's good that you thought to upper case it, but what happens when an individual puts a `-` or a `_` into the page name? 🤔 

- Let's throw an error if `pageTitle` includes a `-` or any empty spaces.
- Let's split the string by `_` to display in the page title variable.
- Let's put the functions which do any string manipulation into `common/utils/string-utils`.

You'll need the same amount of parsing inside the `createPage` function so the file itself isn't created.

This is a great start!